internal/tiltfile/tiltfile_state.go

```diff
@@ -1410,7 +1410,10 @@ func (s *tiltfileState) translateLocal() ([]model.Manifest, error) {
 		}
 
 		paths := append(r.deps, r.workdir)
-		lt := model.NewLocalTarget(model.TargetName(r.name), r.cmd, r.workdir, r.deps).WithRepos(reposForPaths(paths))
+		ignores := s.dockerignoresFromPathsAndContextFilters(paths, r.ignores, r.onlys)
```

Comment:
i don't think this is right?

this reads off all .dockerignores in `paths`, which is not what you want

also, ignores should be evaluated relative to workdir, not relative to deps dirs

Reply:
can you add a test that
1) local_resource does NOT respect .dockerignore, and
2) your ignore pattern is evaluated relative to the correct directory